pkg/resource/validations/kms_arn_regex_validation_test.go

```diff
@@ -41,6 +41,17 @@ var _ = Describe("Validations", func() {
 			})
 		})
 
+		Context("when kmsKeyARN is not empty but is not prefixed with 'mrk'", func() {
+			BeforeEach(func() {
```

Comment:
We only have one test case in this context, is it worth having the `BeforeEach`?